### PR TITLE
#166240939 Setup Redux

### PR DESCRIPTION
**Pull request by eliemugenzi**
### what does this PR do?
- Setup and a simple implementation of Redux
### Description of a task to be completed
- create a redux folder structure
- Install redux-related dependencies
- Create a redux store and reducers
- Create simple actions and action types
### How should it be manually tested?
- Install Redux Devtools chrome extension and inspect the app through it.
### Any background context you want to provide?
- Everyone has to create actions and reducers according to the feature he is working on, but the boilerplate is set up for you.
### What are the relevant pivotal tracker stories?
- [#166240939](https://www.pivotaltracker.com/story/show/166240939)
### Screenshots

Screen Shot 2019-07-29 at 19 41 14

### CODE REVIEW

**Review by mifeille — APPROVED** [review]

**Review by EmyRukundo — APPROVED** [review]

**Review by Diama1 — APPROVED** [review]

**Review by rukundoeric — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -36,6 +36,10 @@
   "dependencies": {
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-router-dom": "^5.0.1"
+    "react-router-dom": "^5.0.1",
+    "react-redux": "^7.1.0",
+    "redux": "^4.0.4",
+    "redux-devtools-extension": "^2.13.8",
```
Comment (deschantkn) [inline package.json:42]: Is this not a dev dependency?

**Review by Gidraf — CHANGES_REQUESTED** [review]
Good work @eliemugenzi but I would suggest you create a folder called assets where all the images, style and fonts will be stored.
I will also suggest you create a utility folder where all project utilities like CONSTANT Strings will be stored, also create tests folder where all the test will be stored.

File: package.json (hunk 1 above)
Comment (eliemugenzi) [reply package.json:42]: It is not a dev dependency, even if it has that `devtools` keyword

**Comment by eliemugenzi** [comment]
> Good work @eliemugenzi but I would suggest you create a folder called assets where all the images, style and fonts will be stored.
> I will also suggest you create a utility folder where all project utilities like CONSTANT Strings will be stored, also create tests folder where all the test will be stored.



@Gidraf DONE.

**Review by Gidraf — APPROVED** [review]

**Comment by Gidraf** [comment]
LGTM

**Pull request merged**